sycl/include/CL/sycl/handler.hpp

```diff
@@ -490,6 +490,8 @@ class __SYCL_EXPORT handler {
     // Empty name indicates that the compilation happens without integration
     // header, so don't perform things that require it.
     if (KI::getName() != nullptr && KI::getName()[0] != '\0') {
+      // TODO support ESIMD in no-integration-header case too.
+      MIsESIMD = KI::isESIMD();
```

Comment:
Looks like implementation of isESIMD() is missing. Should it be here or in another PR?

Reply:
Yes, it's in another PR: https://github.com/intel/llvm/pull/2747/commits/d4f5813b255823f54b6428514b81a1e5be448105#diff-a07ededf6f50832d6f6e956321f97097ff85197e06a3edc9063af88f02d467e2R3836